app/src/main/java/com/example/codingtest1/ContactsAdapter.kt

```diff
@@ -53,16 +58,52 @@ class ContactsAdapter(var contactList: ArrayList<ContactsData>,  private val sha
         popUpMenus.setOnMenuItemClickListener {
             when (it.itemId) {
                 R.id.editText -> {
-                    Toast.makeText(view.context, "Edit text button is clicked", Toast.LENGTH_LONG)
+                    val inflater = LayoutInflater.from(view.context)
+                    val editView = inflater.inflate(R.layout.add_contact, null)
+                    val editName = editView.findViewById<EditText>(R.id.etName)
```

Comment:
You can get rid of multiple findViewById() calls within the adapter by using data binding for adapter as well